openjpa-all/pom.xml

```diff
@@ -31,7 +31,6 @@
         <version>3.0.1-SNAPSHOT</version>
     </parent>
 
-    <groupId>org.apache.openjpa</groupId>
```

Comment:
This change is not related to the topic, I don't see the reason to include it here, please revert.

Reply:
Maven raises the warning: `groupId` is duplicated
I'll move all these changes to separate PR